api/Validators/ClientValidator.cs

```diff
@@ -0,0 +1,17 @@
+using FluentValidation;
+
+public class ClientValidator : AbstractValidator<Client>
+{
+    public ClientValidator()
+    {
+        RuleFor(client => client.Name)
+            .NotNull().WithMessage("Name is required.")
+            .NotEmpty().WithMessage("Name cannot be empty.");
+        RuleFor(client => client.ContactId)
+            .NotNull().WithMessage("Contact ID is required.")
+            .NotEmpty().WithMessage("Contact ID cannot be empty.");
+        RuleFor(client => client.AddressId)
+            .NotNull().WithMessage("Address ID is required.")
+            .NotEmpty().WithMessage("Address ID cannot be empty.");
```

Comment:
De check op zowel contact_id en address_id is goed. Maar ik zou de text naar een passende beschrijving veranderen naar bijvoorbeeld: contact information not filled please provide etc.... (vraag GPT voor een goede beschrijving)

Reply:
Ik mis hier ook de check als de ContactId en de AddressId daadwerkelijk bestaat in de DB. Kijk naar ItemValidator voor inspiratie